### PR TITLE
Quick fix of the docker run command.

### LINKED ISSUE #558: Installation -> Docker uses --user flag which causes container to not be able to create groups
The tutorial in the forum as well as the manual itself reference the --user switch which causes the current container image to fail to start (not able to create group).

    docker run \
        --name openhab \
        --net=host \
        -v /etc/localtime:/etc/localtime:ro \
        -v /etc/timezone:/etc/timezone:ro \
        -v /opt/openhab/conf:/openhab/conf \
        -v /opt/openhab/userdata:/openhab/userdata \
        -v /opt/openhab/addons:/openhab/addons\
        -d \
        --user=<uid> \
        --restart=always \
        openhab/openhab:<version>-<architecture>-<distributions>

This is the error from the docker logs:

    NEW_USER_ID=1008
    NEW_GROUP_ID=1008
    echo 'Starting with openhab user id: 1008 and group id: 1008’
    Starting with openhab user id: 1008 and group id: 1008
    id -u openhab
    Create group openhab with id 1008
    echo ‘Create group openhab with id 1008’
    groupadd -g 1008 openhab
    groupadd: Permission denied.
    groupadd: cannot lock /etc/group; try again later.

I supplied the USER_ID and the GROUP_ID environment variables which were correctly picked up. Removing the --user flag caused the container to start correctly and the files created on the volumes had the correct user ID (1008 in my case).

I suggest changing the documents to:

    docker run \
        --name openhab \
        --net=host \
        -v /etc/localtime:/etc/localtime:ro \
        -v /etc/timezone:/etc/timezone:ro \
        -v /opt/openhab/conf:/openhab/conf \
        -v /opt/openhab/userdata:/openhab/userdata \
        -v /opt/openhab/addons:/openhab/addons\
        -d \
        -e USER_ID=<uid> \
        -e GROUP_ID=<gid> \
        --restart=always \
        openhab/openhab:<version>-<architecture>-<distributions>

I'm using CentOS with SELinux enabled and openhab 2.1 / amd64 / debian image.

Here is the link to my post: https://community.openhab.org/t/running-openhab-2-in-docker/14267/53?u=mathiasringhof

### PR DESCRIPTION
**Pull request by Confectrician**
Closes #558 

Related community post:
https://community.openhab.org/t/running-openhab-2-in-docker/14267/53?u=mathiasringhof

Signed-off-by: Jerome Luckenbach <github@luckenba.ch>

### CODE REVIEW

**Comment by Confectrician** [comment]
I just wanted to address the changes mentioned in the issue @cniweb.

Do you think we should change some more things in the chapter corresponding to this?
I am not using docker and can't evaluate if there is a need for it.

File: installation/docker.md [hunk 1]
```diff
@@ -93,12 +93,13 @@ docker run \
...
         -d \
-        --user=<uid> \
+        -e USER_ID=<uid> \
+        -e GROUP_ID=<gid> \
         --restart=always \
         openhab/openhab:<version>-<architecture>-<distributions>
 ```
 
-Where `<uid>` is the user ID number for the `openhab` user which you can obtain using the command `id openhab`, `<version>` is the version of openHAB, `<architecture>` is the architecture of your system and `<distributions>` is the base system (debian or alpine).
+Where `<uid>` is the user ID number for the `openhab` user which you can obtain using the command `id openhab`, `<gid>` is the group ID number for the `openhab` user, `<version>` is the version of openHAB, `<architecture>` is the architecture of your system and `<distributions>` is the base system (debian or alpine).
```
Comment (bgilmer77) [inline installation/docker.md]: This long sentence might be difficult for a non-English speaking person to understand.  Consider either a list, or four separate sentences.

**Comment by bgilmer77** [comment]
I do not use docker either, so I cannot comment on the technical aspects of this PR.

File: installation/docker.md (hunk 1 above)
Comment (Confectrician) [reply installation/docker.md]: Totally right.
Will change it.

Comment (cniweb) [reply installation/docker.md]: That's exactly what I mean.
I will rebuild the documentation soon anyway.
Technically, the change is OK.

Comment (Confectrician) [reply installation/docker.md]: Ah ok.
Sorry then I misunderstood that.

Comment (Confectrician) [reply installation/docker.md]: Way better now:
image

**Comment by bgilmer77** [comment]
It looks much better as a list to me.  Thank you.

I hesitate to be the one to approve the merge since I do not use docker and cannot say whether the technical changes are correct.  So...

@ThomDietrich are you in a position to approve this?

**Comment by Confectrician** [comment]
See: https://github.com/openhab/openhab-docs/pull/559#discussion_r150685408
> Technically, the change is OK.

LGTM 😃

**Pull request merged**